deploy/helm/edot-collector/kube-stack/managed_otlp/values.yaml

```diff
@@ -522,7 +522,14 @@ collectors:
   #  forwarding telemetry to an Elasticsearch endpoint.
   gateway:
     suffix: gateway
-    replicas: 1
+    replicas: 2
```

Comment:
Just to confirm - Gateway collector also has HPA enabled by default given we are using deployment ?

Reply:
That's a great point, HPA is disabled by default, but I think it would be a better approach for this gateway collector. Compared to the Elasticsearch gateway it just forwards the received signals, IMO a single replica should be enough for non-big clusters. For all the tests we have performed, a single replica has been enough, thus adding another replica might reserve unneeded cluster resources.

Reply:
autoscaling enabled https://github.com/elastic/elastic-agent/pull/6630/commits/918dda25a27b9c8f6370854e75e7e8680249a097

Thanks for pointing that out!